content/docs/services/relational-database.md

```diff
@@ -149,7 +146,11 @@ dump.
 $ pg_restore --clean --no-owner --no-acl --dbname={database name} backup.pg
 ```
 
-### Rotating credentials
+## Encryption
+
+Every RDS instance configured through cloud.gov is [encrypted at rest](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html).
```

Comment:
@mogul Interesting idea, but I think this need would be better served by adding a note saying something like "For details, see SC-28 in the cloud.gov SSP in our [P-ATO documentation package](https://cloud.gov/overview/security/fedramp-tracker/#how-you-can-use-this-p-ato)." Checking whether a config setting does what it's supposed to do is really hard for an outside assessor, but assessors should be able to trust the P-ATO documentation package since it has the 3PAO and JAB review.